### PR TITLE
Add SecureDrop 2.4.0~rc1 changelog

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
## Status

Ready for review
## Description of Changes

Adds 2.4.0 changelog.

## Testing

- [ ] CI is passing
- [x] release version is 2.4.0-rc1
- [x] changelog looks correct.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,41 @@
...
+* Updated datetime formatting to use custom encoder (#6260)
+
+### Operations
+
+* Added support for Tails 5.0 to Journalist and Admin Workstations (#6408, #6424)
+* Disabled fwupd timers (#6401)
+
+### Development
+
+* Fixed schema comparison tests (#6353) (drop?)
```
Comment (eaon) [inline changelog.md]: I assume the `(drop?)` here is an oversight?

Comment (zenmonkeykstop) [reply changelog.md]: yess dammit

Comment (zenmonkeykstop) [reply changelog.md]: (fixed)

**Review by eaon — APPROVED** [review]
Looks good, pending CI passing

**Pull request merged**